code-submissions-clustering-core/src/main/kotlin/org/jetbrains/research/code/submissions/clustering/load/distance/measurers/gumtree/GumTreeDistanceMeasurer.kt

```diff
@@ -48,3 +50,10 @@ class GumTreeDistanceMeasurerByPsi(
             getTreeContext(it)
         }
 }
+
+private fun calculateWeightByAction(action: Action): Int {
+    if (action.instanceOf(Move::class)) {
```

Comment:
Why did you not use: `action is Move`?

Reply:
Probably it is better move this function into GumTreeDistanceMeasurerBase class, what do you think?